tests/server/tso/manager_test.go

```diff
@@ -237,5 +212,5 @@ func waitAllocatorPriorityCheck(cluster *tests.TestCluster) {
 	wg.Wait()
 	// Because the leader changing may take quite a long period,
 	// so we sleep longer here to wait.
-	time.Sleep(waitAllocatorPriorityCheckInterval)
+	time.Sleep(time.Minute)
```

Comment:
can we make it shorter?

Reply:
I think it could be shorter after #3387 merges.

Reply:
After some testing, it turns out we can't make this become shorter...